sycl/source/detail/graph_impl.cpp

```diff
@@ -991,7 +991,7 @@ exec_graph_impl::enqueue(const std::shared_ptr<sycl::detail::queue_impl> &Queue,
               nullptr,
               // TODO: Extract from handler
               UR_KERNEL_CACHE_CONFIG_DEFAULT, CG->MKernelIsCooperative,
-              CG->MKernelUsesClusterLaunch);
+              CG->MKernelUsesClusterLaunch, CG->MKernelWorkGroupMemorySize);
```

Comment:
Could we add a test using work_group_static with Graphs.

If the extensions aren't compatible right now we should be throwing an exception. tested similarly to https://github.com/intel/llvm/blob/sycl/sycl/unittests/Extensions/CommandGraph/Exceptions.cpp#L601 using this handler helper https://github.com/intel/llvm/blob/c35b6ffa401206aa34abe0b59a3aa60a95a8448a/sycl/include/sycl/handler.hpp#L3690

If this does work with graphs then there should be an E2E test in https://github.com/intel/llvm/tree/sycl/sycl/test-e2e/Graph